ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -0,0 +1,206 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const flailDirections = {
+  l: Outputs.left,
+  r: Outputs.right,
+  combo: {
+    en: '${first} => ${second}',
+    de: '${first} => ${second}',
+    fr: '${first} => ${second}',
+    ja: '${first} => ${second}',
+    cn: '${first} => ${second}',
+    ko: '${first} => ${second}',
+  },
+};
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFirstCircleSavage,
+  timelineFile: 'p1s.txt',
+  timelineTriggers: [
+    {
+      id: 'P1S Tile Positions',
+      regex: /(?:First|Second|Third) Element/,
+      beforeSeconds: 3,
+      infoText: (_data, _matches, output) => output.positions!(),
+      outputStrings: {
+        positions: {
+          en: 'Tile Positions',
+        },
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'P1S Warder\'s Wrath',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '662A', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '662A', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '662A', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '662A', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Shining Cells',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6616', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6616', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6616', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6616', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Slam Shut',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6617', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6617', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6617', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6617', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Gaoler\'s Flail Right => Left',
```

Comment:
I think this is flipped, where it meant to say Left => Right.

Reply:
Pretty sure this one is correct. I'm going back in shortly on my main and will confirm then.